### DISCUSSION TITLE
Absolutely atrocious linux experience! Would not consider linux compatible

### DISCUSSION THREAD

**Discussion by brittanpeel** [OrcaSlicer/OrcaSlicer#2115] [General]
At time of installation I am running Fedora 38. I spent just shy of an hour and a half trying to get the appimage running, and once I did it just terribly. I couldn't log in with a google account, so I had to make a bambu account and reconnect my printer(not sure if its a orca problem or bambu problem). Just absolutely atrocious user experience, its obvious that linux support was an after thought. For any linux users that want to use this program, save yourself time and pain and just use a windows vm. 

SHOULD NOT BE CONSIDERED LINUX SUPPORTED!!!

**Comment by kss144** [comment]
I have been using Orcaslicer on ubuntu 22 for quite some time now. I have not faced any problem thus far.
